tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -164,11 +164,11 @@ public async Task<StartResult> StartAsync(ValidatorConfiguration configuration)
         /// Stop monitoring of events.
         /// </summary>
         /// <returns></returns>
-        public async Task<StopResult> StopAsync()
+        public Task<StopResult> StopAsync()
```

Comment:
Shouldn't we rename the method to reflect async removal?

Reply:
The method return a task object, therefor from caller side it is still asynchronously. the `async` keyword is only syntactic sugar. Therefor I wouldn't rename the method